audit-ci.json

```diff
@@ -125,6 +125,13 @@
         "notes": "issue in 'glob' package used by multiple packages (all on different versions) - need to wait for those to upgrade",
         "expiry": "27 November 2025"
       }
+    },
+    {
+      "GHSA-wqch-xfxh-vrr4": {
```

Comment:
This issue has been patched as of yesterday.

https://github.com/expressjs/body-parser/security/advisories/GHSA-wqch-xfxh-vrr4